### DISCUSSION TITLE
Importing CSV

### DISCUSSION THREAD

**Discussion by bholmesaco** [sheltermanager/asm3#1566] [Q&A / Help]
Good morning all, just got Shelter Manager installed on my server and have been enjoying digging through all of the functionality. 

I am having an issue with importing my tickets that were created in a spreadsheet. I have been able to successfully map thing like DISPATCHADDRESS, INCIDENTNOTES and other fields, but there seems to be several missing from the key. How do I map the dispatch times, dates, mileage etc. 

Also, is there already a report to capture ACO mileage by municipality covered, or is that going to take some SQL crafting?

thanks all in advance. 

Anonymized spreadsheet for reference 
https://docs.google.com/spreadsheets/d/14zfTjlshF2Lcq2gnwlz8Vp9xNY0T2HBVmPgGW3hdvmU/edit

**Comment by bholmesaco** [comment]
Most interested in trying to get these to populate from the CSV

Dispatched ACO | 
Dispatch Date/Time |  
Responded Date/Time |  
Followup Date/Time
Completion Date/Time |  
Completion Type

**Comment by Tetzor** [comment]
Hi, imports to those fields aren't currently supported, but we have added a development ticket to include support for them in a future release.

Jon
